releasePipeline/03-repo-branches-delete.sh

```diff
@@ -90,108 +89,74 @@ function ask_user_for_release_type {
     echo "Chosen type of release process: ${release_type}"
 }
 
+function delete_branches {
 
-function set_kubernetes_context {
-    namespace="galasa-build"
-    h1 "Setting the kubernetes context to be cicsk8s, using namespace ${namespace}"
-    kubectl config set-context cicsk8s --namespace=${namespace}
-    rc=$?
-    if [[ "${rc}" != "0" ]]; then 
-        error "Failed. rc=${rc}"
-        exit 1
-    fi
-    success "Kubernetes context set to cicsk8s using namespace ${namespace}"
-}
+    h1 "Deleting all branches in github called ${release_type}"
 
+    branch_name="${release_type}"
 
+    github_username=$(gh api user --jq '.login')
 
-function delete_branches {
+    if [[ $? != 0 ]]; then
+        error "Failed to get the github username. $?"
+        exit 1
+    fi
 
-    h1 "Deleting all branches in github called ${release_type}"
+    workflow_dispatch=$( gh workflow run branch-delete-all --repo ${github_username}/automation --ref main --field distBranch=${branch_name})
 
-    branch_name="${release_type}"
+    if [[ $? != 0 ]]; then
+        error "Failed to call the workflow. $?"
+        exit 1
+    fi
 
-    rm -f temp/delete_branches.yaml
-    cat << EOF > temp/delete_branches.yaml
+    sleep 3
 
-#
-# Copyright contributors to the Galasa project
-#
-# SPDX-License-Identifier: EPL-2.0
-#
+    run_id=$(gh run list --repo ${github_username}/automation --user ${github_username} --limit 1 --json  databaseId --jq '.[0].databaseId')
```

Comment:
Probably best to filter this by `--workflow` instead of `--user` here so we're definitely getting the right workflow. Also will be needed to get rid of the `github_username` variable.